apps/avatax/bruno/environments/localhost.bru

```diff
@@ -0,0 +1,9 @@
+vars {
```

Comment:
**Question:** do we need this env? It has different vars than `staging.bru`

If not maybe we can have `default` env instead those two?